docs/SPEC.md

```diff
@@ -1024,7 +1024,7 @@ Flux source is organized into packages.
 A package consists of one or more source files.
 Each source file is parsed individually and composed into a single package.
 
-    File = [ PackageClause ] [ ImportList ] StatementList .
+    File = [ PackageClause ] [ EditionClause ] [ ImportList ] StatementList .
```

Comment:
Should metadata be before or after the package clause?

I think we might also want this to be more general in the future (a metadata list), but I'm fine with keeping this as specific as possible for now.

Reply:
I think re discussion above it makes sense to be above.